server/v2/api/rest/config.go

```diff
@@ -0,0 +1,18 @@
+package rest
+
+func DefaultConfig() *Config {
+	return &Config{
+		Enable:  true,
+		Address: "localhost:8080",
+	}
+}
+
+type CfgOption func(*Config)
+
+// Config defines configuration for the HTTP server.
+type Config struct {
+	// Enable defines if the HTTP server should be enabled.
```

Comment:
can we say rest server instead of http server? so it matches the component name.